cogs/onmyoji.py

```diff
@@ -153,18 +269,100 @@ def location_finder(self, shiki):
         search = ' '.join([term.lower() for term in search])
         for shiki in self.shikigami_class.keys():
             if search in shiki:
-                await ctx.send(self.shiki_found(shiki))
-                await ctx.send(self.location_finder(shiki))
+                shiki_embed, shiki_icon = self.shiki_bounty_embed(shiki)
+                await ctx.send(file=shiki_icon, embed=shiki_embed)
                 return
             if search in self.shikigami_class[shiki].hints.lower():
-                await ctx.send(self.shiki_found(shiki))
-                await ctx.send(self.location_finder(shiki))
+                shiki_embed, shiki_icon = self.shiki_bounty_embed(shiki)
+                await ctx.send(file=shiki_icon, embed=shiki_embed)
+                return
+            if search in self.shikigami_class[shiki].alias.lower():
+                shiki_embed, shiki_icon = self.shiki_bounty_embed(shiki)
+                await ctx.send(file=shiki_icon, embed=shiki_embed)
+                return
+        await ctx.send("For all my bath powers, I could not find your term, or something went wrong.")
+
+    @commands.command()
+    async def bbt(self, ctx):
+        name = 'Orochi'
+        print(name)
+        with open(config.bbt_csv_db_file, newline='') as bbt_db:
+            bbt_db_reader = csv.reader(bbt_db)
+            for i in range(0, 6):
+                next(bbt_db_reader)
+            bbt_db = [row for row in bbt_db_reader]
+        locale_list = []
+        for row in bbt_db:
+            if name.lower() in row[2].lower():
+                contains = row[2].split('\n')
+                temp_list = []
+                temp_list.append(row[0])
+                temp_list.append(row[1])
+                for each in contains:
+                    if name.lower() in each.lower():
+                        temp_list.append(each)
+                        break
+                locale_list.append(temp_list)
+        if len(locale_list)==0:
+            print('Entry not found in DB.')
+            return
+        main_sub_and_shiki_list = []
```

Comment:
This section looks identical to `generate_bbt_locations` should probably reuse the function.

Reply:
oh this was dummy code to test the function